src/ngx-smart-modal/src/components/ngx-smart-modal.component.ts

```diff
@@ -71,9 +74,13 @@ export class NgxSmartModalComponent implements OnInit, OnDestroy {
 
   private _data: any;
 
-  @ViewChildren('nsmContent') private nsmContent: QueryList<ElementRef>;
-  @ViewChildren('nsmDialog') private nsmDialog: QueryList<ElementRef>;
-  @ViewChildren('nsmOverlay') private nsmOverlay: QueryList<ElementRef>;
+  private positionX = 0;
+  private positionY = 0;
+  private dragging = false;
+
+  @ViewChildren('nsmContent') private nsmContent!: QueryList<ElementRef>;
+  @ViewChildren('nsmDialog') private nsmDialog!: QueryList<ElementRef>;
+  @ViewChildren('nsmOverlay') private nsmOverlay!: QueryList<ElementRef>;
```

Comment:
Pull the master and rebase your branch please. Now there is no `!` after name of variables. Like you deleted